pxf/pxf-service/src/scripts/pxf-env.sh

```diff
@@ -34,7 +34,11 @@ export PXF_LOGDIR=@pxfLogDir@
 export PXF_RUNDIR=@pxfRunDir@
 
 # Configured user
-export PXF_USER=${PXF_USER:-@pxfDefaultUser@}
+if [ ! -z "@pxfDefaultUser@" ]; then
```

Comment:
these conditions will look really weird in the generated script, would it be better to assign this to variable, which gets filtered value and then check the value of that variable in all the if statements.